content/en/docs/languages/go/instrumentation.md

```diff
@@ -690,12 +711,37 @@ func init() {
 
 ### Adding attributes
 
+Attributes are keys and values that are applied as metadata to your metrics and
+are useful for aggregating, filtering, and grouping metrics. Attributes can be
+added at instrument creation, or at any other time during the lifecycle of an
+instrument.
```

Comment:
```suggestion
[Attributes] are keys-values pairs applied as metadata to your metrics that
are useful for aggregating, filtering, and grouping metrics. Attributes can be
added at instrument creation, or at any other time during the lifecycle of an
instrument.

[Attributes]: /docs/concepts/signals/traces/#attributes
```

Reply:
s/key-values/key-value

Reply:
I decided only to take this for the traces section, since the link takes you to the traces signal page.